webapp/src/components/ExchangePage/index.jsx

```diff
@@ -41,6 +41,8 @@ import './ExchangePage.css'
 
 const readWeb3 = new Web3(process.env.REACT_APP_NETWORK_URL)
 
+const EXECUTE_ORDER_GAS_USAGE = '300000' // 300,000 GAS
```

Comment:
On [this line](https://github.com/UniswapEx/exchange/pull/54/files#diff-21c64e0852ec5eb78d274c7c93279cbdL71) we are using `300000`. What do you think of splitting that reuse this var there?